src/tests/FunctionalTests/Android/Device_Emulator/gRPC/grpc-dotnet/testassets/Certs/InteropTests/server1.key

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
this needs to be added to https://github.com/dotnet/runtime/blob/main/.config/CredScanSuppressions.json otherwise CredScan will complain about leaking this credential